src/Dapr.Cryptography/Encryption/Models/DecryptionOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ------------------------------------------------------------------------
+//  Copyright 2025 The Dapr Authors
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  ------------------------------------------------------------------------
+
+namespace Dapr.Cryptography.Encryption.Models;
+
+/// <summary>
+/// A collection fo options used to configure how decryption cryptographic operations are performed.
+/// </summary>
+public sealed class DecryptionOptions
+{
+    private int streamingBlockSizeInBytes = 4 * 1024; // 4KB
```

Comment:
4KB seems a little small for a buffer, but not 100% sure what the .NET gRPC defaults are (16KB? 64KB?).

Reply:
Anyone is welcome to change this to match their own situations, but this mirrors the default file stream buffer size in C# (e.g. via `FileStream`).